providers/mandrill/src/lib/mandrill.provider.ts

```diff
@@ -64,13 +64,21 @@ export class MandrillProvider implements IEmailProvider {
     };
   }
 
-  async checkIntegration(
-    options: IEmailOptions
-  ): Promise<ICheckIntegrationResponse> {
-    return {
-      success: true,
-      message: 'Integrated successfully!',
-      code: CheckIntegrationResponseEnum.SUCCESS,
-    };
+  async checkIntegration(): Promise<ICheckIntegrationResponse> {
+    try {
+      await this.transporter.users.ping();
+
+      return {
+        success: true,
+        message: 'Integrated successfully!',
+        code: CheckIntegrationResponseEnum.SUCCESS,
+      };
+    } catch (error) {
+      return {
+        success: false,
+        message: error?.message,
```

Comment:
can we have better error mapping, similar to the one you did in the previous PR?

Reply:
Unfortunately I searched through the entire API documentation (https://mailchimp.com/developer/transactional/api/) and [mailchimp-transactional](https://github.com/mailchimp/mailchimp-transactional-node/blob/master/src/ApiClient.js#L70-L104) source code (it is simply using [axios](https://axios-http.com/)), and could not find any error code mapping

Reply:
https://mailchimp.com/developer/marketing/docs/errors/#error-glossary
is this helpful?
i think 401 can be mapped

Reply:
That documentation is for the Marketing API though, not the Transactional Email API, they are 2 totally different products

Reply:
ok thought so,
Alright then let's leave it as it is
Thank you 🚀